### PR TITLE
Added background gradient for fast-scrolling (closes #26)

### LINKED ISSUE #26: Add background gradient for fast-scrolling

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
Seems to add ~50-80ms to a refresh tick. This patch applies the gradient in phase2, so this won't block first paint or content being interactive.

**Comment by wilsonpage** [comment]
@etiennesegonzac r?

**Comment by etiennesegonzac** [comment]
<3 the choice of gradient :)
Looks like we need to offset the background though for the "search field" use case.
(otherwise it looks like the field is fast scrolling :))

**Comment by wilsonpage** [comment]
@etiennesegonzac ah I see :) It's not that easy/possible to offset a repeating gradient is it? The simplest fix would be to apply a background-color to the search component. Any other suggestions?

**Comment by etiennesegonzac** [comment]
I wonder if a subtle use of `background-position` and `repeating-linear-gradient` could help, but applying a background color to the search component sounds good too :)

File: gaia-fast-list.js [hunk 1]
```diff
@@ -1058,9 +1031,12 @@ Internal.prototype = {
...
     var top = this.el.top;
 
-    return (top != null && bottom != null)
-      ? parent.innerHeight - top - bottom
-      : this.el.clientHeight;
+    if (top != null && bottom != null) {
+      return parent.innerHeight - top - bottom;
+    }
+
+    return parseInt(this.el.style.height)
```
Comment (wilsonpage) [inline gaia-fast-list.js:1038]: This avoids a reflow on startup if the component has `style.height`. Sorta unrelated but ... meh.

**Comment by wilsonpage** [comment]
@etiennesegonzac fixed the offset issue by programmatically drawing the gradient. Tested too.

**Comment by wilsonpage** [comment]
Happy to merge if you are :green_heart:

**Comment by etiennesegonzac** [comment]
Ship it!

**Comment by wilsonpage** [comment]
:ship: ed

**Comment by etiennesegonzac** [comment]
:)

**Pull request merged**